index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@react-native/assets-registry/registry';
```

Comment:
Unfortunately, without it, we have a problem with typescript

Reply:
You answered only the first question here 😅

Reply:
@WoLewicki Ah, you're right sorry. Hm... I'm a little confused because I'm not sure how we build the library 🧐

Reply:
You can run `npm pack` to get the package that would be available on `npm`. Then you can add this package to a new project to test it locally. Is it what you meant?

Reply:
@WoLewicki, I compiled the package, added it to the empty project, and everything worked well. 
Without any extra changes.

Reply:
nope, I didn't change the package.json file.
as I understand if we want to add that file we have to add them to package.json in section files, yes?

Reply:
Maybe it would be easier to have that file in the project because it isn't included in the building process and doesn't change anything. But it's only my suggestion :D

Reply:
after a discussion about that with @jakex7  we decided to remove the ndex.d.ts file.